testnets/neutrontestnet/chain.json

```diff
@@ -25,16 +25,16 @@
   },
   "codebase": {
     "git_repo": "https://github.com/neutron-org/neutron",
-    "recommended_version": "v4.0.0-rc3",
+    "recommended_version": "v4.2.2-testnet",
     "compatible_versions": [
-      "v4.0.0-rc3"
+      "v4.2.2-testnet"
     ],
     "cosmos_sdk_version": "0.50",
     "consensus": {
       "type": "cometbft",
-      "version": "0.38.7"
+      "version": "0.38.11"
     },
-    "cosmwasm_version": "0.52",
+    "cosmwasm_version": "0.51",
```

Comment:
Will need to update the `cosmwasm` property as well

Reply:
done as well. please take another look 🙏  it must be all